Src/Support/GoogleApis/Apis/Util/Store/NullDataStore.cs

```diff
@@ -0,0 +1,67 @@
+﻿/*
+Copyright 2017 Google Inc
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+using Google.Apis.Util.Store;
+using System.Threading.Tasks;
+
+namespace GoogleApis.Apis.Util.Store
+{
+    /// <summary>
+    /// A null datastore. Nothing is stored, nothing is retrievable.
+    /// </summary>
+    class NullDataStore : IDataStore
+    {
+        /// <summary>
+        /// Construct a new null datastore, that stores nothing.
+        /// </summary>
+        public NullDataStore()
+        {
+        }
+
+        private Task<T> CompletedTask<T>()
```

Comment:
Can we not use `Task.FromResult` due to the frameworks we're targeting? Even if we can't, we could put this in a static readonly field and return the same task everywhere.